static/js/home.js

```diff
@@ -120,27 +141,11 @@ function getEventCB(data){
   } else {
     getAccidentMarkerCB(homeJSLocal.map, data.data);
   }
-}
 
-// TO DO plot the actual data instead of the dummy data.
-function drawTableEvent() {
-      google.charts.setOnLoadCallback(drawTable);
-      function drawTable() {
-        var data = new google.visualization.DataTable();
-        data.addColumn('string', 'Name');
-        data.addColumn('number', 'Salary');
-        data.addColumn('boolean', 'Full Time Employee');
-        data.addRows([
-          ['Mike',  {v: 10000, f: '$10,000'}, true],
-          ['Jim',   {v:8000,   f: '$8,000'},  false],
-          ['Alice', {v: 12500, f: '$12,500'}, true],
-          ['Bob',   {v: 7000,  f: '$7,000'},  true]
-        ]);
-
-        var table = new google.visualization.Table(document.getElementById('table_div'));
-
-        table.draw(data, {showRowNumber: true, width: '100%', height: '20%'});
-      }
+  // udpate the table
+  if( homeJSLocal.detailViewState == 'on'){
```

Comment:
Is that possible to change homeJSLocal.detailViewState to a boolean value, so that you can use 
if(homeJSLocal.detailViewState)
to check it